### ISSUE TITLE
模型转换时打包归一化预处理操作 | Maple.Link

### ISSUE THREAD

**Issue by Casxt** [Casxt/casxt.github.io#28]
https://maple.link/2021/12/09/%E6%A8%A1%E5%9E%8B%E8%BD%AC%E6%8D%A2%E6%97%B6%E6%89%93%E5%8C%85%E5%BD%92%E4%B8%80%E5%8C%96%E9%A2%84%E5%A4%84%E7%90%86%E6%93%8D%E4%BD%9C/ 

 在模型运行时，预处理是一个非常耗时的操作，其实在进行模型转换时就可以将预处理一同打包到模型中，在转换为ONNX或者Tensorrt模型后就可以随着模型一起被加速执行。 归一化Module如下： 123456789class Normalize(torch.nn.Module):    def __init__(self):        super().__init__()        self
